oioswift/common/middleware/copy.py

```diff
@@ -72,19 +74,21 @@ def fast_copy_allowed(self, req):
                              src_container_name,
                              src_obj_name))
         try:
-            obj_inf = get_object_info(req.environ, self.app,
-                                      path=src_path,
-                                      swift_source='SSC')
-            is_slo = obj_inf.get('sysmeta', {}).get('slo-size', False)
+            req = _prepare_pre_auth_info_request(req.environ, src_path, 'SSC')
+            resp = req.get_response(self.app)
+
+            is_slo = resp.headers.get('X-Object-Sysmeta-Slo-Size', False)
```

Comment:
I'm not sure it is available at this point (I saw some low-level cases where it was not), but the proper way to identify a SLO is with `X-Static-Large-Object`.

Reply:
Done, it works with `x-static-large-object`, so let use the proper way